notesystem/modes/convert_mode.py

```diff
@@ -28,6 +28,8 @@ class PandocOptions(TypedDict):
     arguments: Optional[str]
     # The template to be used
     template: Optional[str]
+    # The output format (for now 'html' or 'pdf')
+    output_format: str
```

Comment:
Could be an enum or something like it. To ensure only valid output_format can be added. It would also allow for easy generation of the `--to-x` flags. 

This can be added when it's needed tho. So this is fine for now.